Source/VolunteerReporting/Web.Angular/src/app/core/aggregated-case-report.service.ts

```diff
@@ -8,6 +8,7 @@ import { Headers, Http } from '@angular/http';
 import { Report } from '../shared/models/report.model';
 import { CaseReport } from '../shared/models/index';
 import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/observable/forkJoin'; 
```

Comment:
Might make more sense to just import Observable directly from rxjs, but this can also be fixed later
(This could consolidate a few import statements)
import { Observable } from 'rxjs'

Reply:
Do we close the bug then?

Reply:
I'd say we leave #329 open. This fixes the error, but there's probably a better solution which can clean up these import statements and work as an example on how to do this